knowledge-base/adr/0033-send-email.md

```diff
@@ -145,3 +146,17 @@ Applications on BTP can send mails with `MAIL` destinations to cloud/On-Prem mai
 ### Useful links
 
 - [Using socks proxy with nodemailer](https://nodemailer.com/smtp/proxies/#2-using-socks-proxy)
+
+### PoC notes
+#### Protocol for sending emails
+- using SMTP protocol
+  Working with `nodemailer` + basic auth
+- using socket protocol
+  Our PoC tried `WS.WebSocket`, but as socket protocol without knowing SMTP, it did not support email related APIs.
+  E.g, authentication + email properties (from/to...) cannot be configured
+#### OnPrm Proxy
+- support socket proxy for now
+  It uses custom auth handler with JWT.
+- http proxy (later)
+  Below is one example with `node-http-proxy-agent`, that is not tested, and has cumbersome implementation.
+  [Useful example](https://github.com/TooTallNate/node-http-proxy-agent/blob/master/src/agent.ts#L83)
```

Comment:
[nth] The inclusive plugin is complaining about the 'master' in the path. Perhaps you can somehow ignore this chec.